packages/odyssey-react/src/components/Box/_box-root.scss

```diff
@@ -12,14 +12,9 @@
 
 .root {
   box-sizing: border-box;
-  color: $text-body;
```

Comment:
This would prevent Box from working correctly without child Text components, right?

Reply:
To get around that Box takes Text's utility class (unless you opt out)

Reply:
I'm basically doing what you describe by conditionally applying those defaults from Text's utility classes.

Text opts out of that behavior and applies it's own defaults or the supplied props from a user.

Having to opt out of Box's text defaults felt safer than relying on an override or (further) muddying  Box's API with all those text props.

Reply:
The difficulty that I encountered with just using overrides is that with a strength of 1 source order matters, Like in the case of the storybook example I had to[ double up to override the root color](https://github.com/okta/odyssey/blob/develop/packages/odyssey-storybook/.storybook/preview-head.html#L26). 

It's a little tricky but more sustainable in my mind to switch off the default and add a class.